solution/backend/regulations/templates/regulations/partials/footer.html

```diff
@@ -12,9 +12,9 @@
         
     </div>-->
     <div class="footer-middle match-middle">
-        <h4 class="ds-text-heading--md">Limitations and disclaimers</h4>
-        <p>This tool is an unofficial compilation of publicly available materials provided for informational purposes only. Not comprehensive or error-free; please verify information with original sources. Additional relevant and up-to-date policy information is available from official government websites. This is not an official edition of the <a href="https://www.govinfo.gov/app/collection/cfr" target="_blank" rel="noopener noreferrer" class="external">Code of Federal Regulations</a>.</p>
-        <p>This independent, privately-funded project uses free and open source code developed by U.S. government agencies, but it is not supported by or affiliated with any government agency.</p>
+        <h4 class="ds-text-heading--md">Limitations</h4>
+        <p>This tool is an unofficial compilation of publicly available materials provided for informational purposes only. Not comprehensive or error-free; please verify information with original sources. Additional relevant and up-to-date policy information is available from official government websites. This independent project is not supported by or affiliated with any government agency.</p>
+        <p><em>This site works best on desktop browsers.</em></p>
```

Comment:
Maybe add something like: 

"For the moment, this site works best on desktop browsers." 

to emphasize that we're going to work on responsiveness at some point, just not now.